src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java

```diff
@@ -149,7 +149,31 @@ public class HydrateBreakMessageDictionary {
                         add("Thousands Have Lived Without Love, Not One Without Water. Take a sip");                  
                     }}
             );
-    
+
+    /**
+     * Hydrate Reminder interval break text to display in Aggressive form
+     */
+    private static final List<String> HYDRATE_BREAK_AGGRESSIVE_TEXT_LIST =
+            Collections.unmodifiableList(
+                    new ArrayList<String>() {{
+                        add("Go grab some Water NOW!");
+                        add("Water Now or Never , You need It!");
+                        add("You should be Hydrated , No questions asked!");
+                        add("I need to see you hydrating right here right now with no excuses");
+                        add("Hydration Time! You Gotta Enjoy it");
+                        add("Thirsty? We Know , All you want is Water");
+                        add("Your Body is made up of 70% water , Don't Drain that number to 0 , Go Drink Water");
```

Comment:
Just a small nitpick, can you remove the space before commas in all your messages?

For example `hydrated, no` instead of `hydrated , no`


After you update this PR and resolve the merge conflicts, I'll approve your changes and merge them in. :)

Reply:
I made all the changes you marked and I have also made a merge commit